### ISSUE TITLE
Attempting to use TensorFlow Lite Model Maker, but having problems installing

### ISSUE THREAD

**Issue by chaberfeld** [Kaggle/kaggle-api#425]
Not sure if this is a bug, or just not allowed.
When running this:
 
!pip install -q tflite-model-maker

Returns with the error:

ERROR: Cannot uninstall 'llvmlite'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.

Can TFLMM be used on Kaggle? 
If not, is there a way to redo using Keras?

Thank you,  Carl

**Comment by michaelriedl** [comment]
I was having the same issue and this worked for me:

```bash
!pip install --upgrade pip -q
!pip install tflite-model-maker -q
```

**Comment by chaberfeld** [comment]
Thank you for the suggestion.
I am running the notebook for the Tomato Leaf challenge on the Kaggle site.
After replacing the two lines of code, this error is returned by Kaggle:

ERROR: Cannot uninstall 'llvmlite'. It is a distutils installed
project and thus we cannot accurately determine which files belong to
it which would lead to only a partial uninstall.


Your code works OK on Colab. Appears that TFLMM wants to replace
llvmlite, but Kaggle won't allow this.

**Comment by michaelriedl** [comment]
Here is a notebook on Kaggle with it working:
[TF Lite Model Maker](https://www.kaggle.com/code/michaelriedl/pokemon-code-card-detector-tflite-model-maker)

**Comment by chaberfeld** [comment]
Hi Michael

Thanks for your assistance (and persistence)!

I used the pip statements from the linked notebook, at first without
success, but I saw they used GPU.
When switching to P100, noticed my environment was locked to May 2022. Then
changed to "use the latest".
One/both of these allowed me to load TFLMM, AT LAST. Must be back to my
tomato leaf challenge !

Thank you again for the help,
Carl

Thanks again,  Carl
